### PR TITLE
Add tests for mq

### PR DESCRIPTION
**Pull request by Nyarum**
Work for #86 

Run tests via Makefile (For example):
- `make test_mq`

Output, like:
```
go test -v ./cmd/mq
=== RUN   TestMqCreate
       Creating queue "testQueue"
----->  Queue testQueue has been successfully created.
       Visit hud-e.iron.io/mq/mq-aws-us-east-1-1/projects/57be1db75b0f340007436c8b/queues/testQueue for more info.
--- PASS: TestMqCreate (1.93s)
=== RUN   TestMqClear
-----> Queue testQueue has been successfully cleared
--- PASS: TestMqClear (0.18s)
=== RUN   TestMqPush
6333405418112196614
6333405418112196615
6333405418112196616
6333405418112196617
--- PASS: TestMqPush (0.17s)
=== RUN   TestMqPop
6333405418112196614 "test1"
--- PASS: TestMqPop (0.17s)
=== RUN   TestMqReserve
6333405418112196615 63475e9d057209a6b65b91dba9ef40fe "test2"
--- PASS: TestMqReserve (0.17s)
=== RUN   TestMqPeek
6333405418112196616 "test3"
--- PASS: TestMqPeek (0.17s)
=== RUN   TestMqList
queue2
testQueue
--- PASS: TestMqList (0.19s)
...
```

### CODE REVIEW

**Comment by Nyarum** [comment]
@nikhilm still in progress.

**Comment by Nyarum** [comment]
@nikhilm R4R.

File: cmd/mq/mq_test.go [hunk 1]
```diff
@@ -0,0 +1,161 @@
+package mq
```
Comment (nikhilm) [inline cmd/mq/mq_test.go]: This file can be refactored into one table driven test that just reads the constructor function and args from a table and runs the action and checks that err == nil.

File: cmd/mq/mq_test.go [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+import (
+	"testing"
+
+	"github.com/iron-io/ironcli/common"
+)
+
+func TestMqCreate(t *testing.T) {
+	var (
```
Comment (nikhilm) [inline cmd/mq/mq_test.go]: Why this nested var declaration here and everywhere. Just make it `var settings = ...`

File: tests/test.js [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (nikhilm) [inline tests/test.js]: Extra file.

**Comment by Nyarum** [comment]
@nikhilm fixes is added.

**Pull request merged**